cmd/rpc.go

```diff
@@ -19,24 +23,50 @@ const (
 var (
 	requestURL    string
 	authTokenFlag string
+	storePath     string
 )
 
-func InitURLFlag() (*string, string, string, string) {
-	return &requestURL, "url", defaultRPCAddress, "Request URL"
-}
+func RPCFlags() *flag.FlagSet {
+	fset := &flag.FlagSet{}
+
+	fset.StringVar(
+		&requestURL,
+		"url",
+		defaultRPCAddress,
+		"Request URL",
+	)
 
-func InitAuthTokenFlag() (*string, string, string, string) {
-	return &authTokenFlag,
+	fset.StringVar(
+		&authTokenFlag,
 		"token",
 		"",
-		"Authorization token (if not provided, the " + authEnvKey + " environment variable will be used)"
+		"Authorization token (if not provided, the "+authEnvKey+" environment variable will be used)",
+	)
+
+	storeFlag := NodeFlags().Lookup(nodeStoreFlag)
+
+	fset.StringVar(
+		&storePath,
+		nodeStoreFlag,
+		"",
+		storeFlag.Usage,
+	)
```

Comment:
is there not a better way to import the flags? this looks messy

Reply:
Why is it messy? there is one extra line `storeFlag := NodeFlags().Lookup(nodeStoreFlag)`. All other code has the same approach.

Reply:
reworked